packages/containers/src/GuidedTour/GuidedTour.sagas.js

```diff
@@ -0,0 +1,22 @@
+import { put, take, takeEvery } from 'redux-saga/effects';
+
+import Connected from './GuidedTour.connect';
+import Constants from './GuidedTour.constants';
+
+export function* show() {
+	yield put(Connected.setStateAction({ show: true }));
+}
+
+export function* hide() {
+	yield put(Connected.setStateAction({ show: false }));
+}
+
+function* defaultHandler() {
+	yield takeEvery(Constants.GUIDED_TOUR_SHOW, show);
+	yield takeEvery(Constants.GUIDED_TOUR_HIDE, hide);
+	yield take('DO_NOT_QUIT');
```

Comment:
I'm not sure this last take is required.
If i remember well, redux-saga handle this case by default ( to confirmed )

Reply:
Just a copy/paste, I will try